akka-actor-typed-tests/src/test/scala/akka/actor/typed/scaladsl/StopSpec.scala

```diff
@@ -21,22 +20,49 @@ class StopSpec extends ScalaTestWithActorTestKit with WordSpecLike {
 
   "Stopping an actor" should {
 
-    "execute the post stop" in {
+    "execute the post stop when stopping after setup" in {
       val sawSignal = Promise[Done]()
```

Comment:
why use different approaches in different tests? I'd prefer TestProbe

Reply:
No real motivation, I'll change.